docs/decisions/0012-assignment-based-policies.rst

```diff
@@ -0,0 +1,133 @@
+0012 Assignment-based policies
+*******************************
+
+Status
+======
+
+Accepted - September 2023
+
+Context
+=======
+Some enterprise customers want tighter control over the purse strings, particularly for
```

Comment:
nit: What does "purse strings" mean? Is there a way to say the same thing without feeling too slangy?